voting/src/lib.rs

```diff
@@ -0,0 +1,284 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use near_sdk::collections::Map;
+use near_sdk::{env, near_bindgen, AccountId, Balance, EpochHeight};
+
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+type Timestamp = u64;
```

Comment:
Let's change to `U64` and call it `WrappedTimestamp` to be json friendly. Otherwise modify the actual returned value in `get_result` to be `Option<U64>`
```suggestion
type WrappedTimestamp = U64;
```